sunbird-rc-esignet-integration-impl/src/main/java/io/mosip/esignet/sunbirdrc/integration/service/SunbirdRCVCIssuancePlugin.java

```diff
@@ -0,0 +1,260 @@
+                        //casting it to JsonLD object
+                        VCResult vcResult = new VCResult();
+                        JsonLDObject vcJsonLdObject = JsonLDObject.fromJsonObject(vcResponseMap);
+                        vcResult.setCredential(vcJsonLdObject);
+                        vcResult.setFormat(FORMAT);
+                        return vcResult;
+                    }else{
+                        log.error("Sunbird service is not running. Status Code: " ,responseEntity.getStatusCode());
+                        throw new VCIExchangeException(ErrorConstants.VCI_EXCHANGE_FAILED);
+                    }
+                }catch (Exception e){
+                    log.error("Unable to parse the Registry Object :{}",credentialRequestMap);
+                    throw new VCIExchangeException(ErrorConstants.VCI_EXCHANGE_FAILED);
+                }
+        }else {
+            log.error("Sunbird service is not running. Status Code: " ,responseEntity.getStatusCode());
+            throw new VCIExchangeException(ErrorConstants.VCI_EXCHANGE_FAILED);
+        }
+    }
+
+    @Override
+    public VCResult<String> getVerifiableCredential(VCRequestDto vcRequestDto, String holderId, Map<String, Object> identityDetails) throws VCIExchangeException {
+        return null;
+    }
+
+    private Map<String,Object> createCredentialIssueRequest(String requestedCredentialType,Map<String,Object> responseMap,VCRequestDto vcRequestDto,String holderId) {
+
+        Template template=credentialTypeTemplates.get(requestedCredentialType);
+        Map<String,String> configMap=credentialTypeConfigMap.get(requestedCredentialType);
+        StringWriter wrt = new StringWriter();
+        VelocityContext context = new VelocityContext();
+        Map<String,Object> requestMap=new HashMap<>();
+
+        try{
+            context.put("issuerId", configMap.get(STATIC_VALUE_MAP_ISSUER_ID));
+            context.put("calNowPlus30Days", calculateNowPlus30Days());
+            context.put("id",holderId);
+            for (Map.Entry<String, Object> entry : responseMap.entrySet()) {
+                String key = entry.getKey();
+                Object value = entry.getValue();
+                if (value instanceof List) {
+                    JSONArray jsonArray = new JSONArray((List<String>) value);
+                    context.put(key, jsonArray);
+                } else {
+                    context.put(key, value);
+                }
+            }
+        }catch (Exception e){}
+        template.merge(context, wrt);
+        try{
+            Map<String,Object> tempMap=mapper.readValue(wrt.toString(),Map.class);
+            requestMap.put("credential",tempMap);
+            requestMap.put("credentialSchemaId",configMap.get(CRED_SCHEMA_ID));
+            requestMap.put("credentialSchemaVersion",configMap.get(CRED_SCHEMA_VESRION));
+            requestMap.put("tags",new ArrayList<>());
+        }catch (Exception e){
+        }
```

Comment:
why exception is not logged or thrown out like other exceptions ?

Reply:
added